code/Tasks/LatestWorkableJobsTask.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace SilverStripe\Workable\Tasks;
+
+use Psr\SimpleCache\CacheInterface;
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Workable\Workable;
+
+class LatestWorkableJobsTask extends BuildTask
+{
+    /**
+     * @inheritdoc
+     */
+    protected $title = 'Refresh cache of Workable Jobs';
+
+    /**
+     * @inheritdoc
+     */
+    protected $description = 'Refresh cache of Workable Jobs';
+
+    /**
+     * @inheritdoc
+     */
+    public function run($request)
+    {
+        Workable::flush();
+
+        $params = ['state' => 'published'];
+        $jobs = singleton(Workable::class)->getFullJobs($params);
+
+        $output = "0 jobs to import";
+
+        if ($jobs && $jobs->count()) {
+            $cacheKey = 'FullJobs' . implode('-', $params);
+            $cache = Injector::inst()->get(CacheInterface::class . '.workable');
```

Comment:
Update `Workable` class to include a method that returns the cache and use that. So we have one place where the cache is defined.

Reply:
now using Workable Class getCache method